src/UIComponents/GenerateResults.jsx

```diff
@@ -2,7 +2,6 @@
 import PropTypes from "prop-types";
 import React, { Component } from "react";
 import { connect } from "react-redux";
-import train from "../train";
```

Comment:
@breville this and below were unrelated changes I needed to make the linter happy. Did something unexpected sneak in somewhere? 🤷‍♀️

Reply:
Thanks for fixing this.  I missed a failing check for the final commit to https://github.com/code-dot-org/ml-playground/pull/180.